docs/endpoints/getGJLevelScores211.md

```diff
@@ -1,7 +1,5 @@
 # getGJLevelScores211.php
 
-Upload and query user levelBoard record.
```

Comment:
I did not say remove the entire description but just you throwing the word query for no good reason. follow the format which colon uses